Makefile

```diff
@@ -476,7 +391,7 @@ release-once-tagged:
 	@echo "- Copy the tag content (release notes) shown on that page"
 	@echo "- Go to https://github.com/projectcalico/felix/releases/new?tag=$(VERSION)"
 	@echo "- Paste the copied tag content into the large textbox"
-	@echo "- Attach the PyInstaller bundle"
+	@echo "- Attach the binary"
```

Comment:
I think we do need to attach something; we need it to maintain the bare-metal install guide (which doesn't use debs or RPMs).  The bundle didn't have source code inside, it was pre-compiled python.

github automatically adds a source tarball to the release, which is annoying because that tarball doesn't know what its version number is.